### ISSUE TITLE
OutOfRangeError

### ISSUE THREAD

**Issue by finnick99** [kelkelcheng/GC-Net-Tensorflow#6]
hi
Do I have to generate .tfrecords files to use pre-trained model， Can I download "saved model " to test the new images? I downloaded the "saved model" folder directly but had an error using test.py:
OutOfRangeError (see above for traceback): RandomShuffleQueue '_5_shuffle_batch_1/random_shuffle_queue' is closed and has insufficient elements (requested 1, current size 0)
	 [[Node: shuffle_batch_1 = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_FLOAT, DT_FLOAT], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/cpu:0"](shuffle_batch_1/random_shuffle_queue, shuffle_batch_1/n)]]
My computer configuration cannot support training, can i directly test the images？ 
thanks for your reply,  apologize for my ignorance as a beginner
